packages/cli-main/src/cli/commands/upgrade.ts

```diff
@@ -1,12 +1,13 @@
 import {Flags} from '@oclif/core'
-import {error, file, output, path} from '@shopify/cli-kit'
+import {error, file, os, output, path, system} from '@shopify/cli-kit'
 import {
   addNPMDependencies,
   checkForNewVersion,
   DependencyType,
   getPackageManager,
 } from '@shopify/cli-kit/node/node-package-manager'
 import Command from '@shopify/cli-kit/node/base-command'
+const cliDependency = '@shopify/cli'
```

Comment:
Considering this information is in the `package.json` of the package containing this module, I'd keep that as the source of truth and read the information at runtime.